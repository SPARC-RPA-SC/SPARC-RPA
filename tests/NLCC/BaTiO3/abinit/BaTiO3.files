BaTiO3.in 
BaTiO3.out 
tbase_xi
tbase_xo 
tbase_x 
./../../../../psps_LDA/Ba.psp8
./../../../../psps_LDA/Ti.psp8
./../../../../psps_LDA/O.psp8

