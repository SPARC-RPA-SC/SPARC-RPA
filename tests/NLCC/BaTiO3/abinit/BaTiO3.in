chksymbreak 0 
prtwf 0 
kptopt 1
ixc 2
shiftk 0.0 0.0 0.0 
occopt 7 
tsmear 0.007349864
toldfe 1.0d-8
ngkpt 1 1 1
nband 29
rprim 
1 0 0
0 1 0
0 0 1

acell 
7.630000 7.630000 7.630000 

natom 
5

ntypat 
3

znucl 
56 22 8

typat 
1 2 3 3 3

xcart
7.540463013371621   7.248833728543416   7.518607464589964
3.725768164762889   3.878328962888697   3.426961342823767
4.052740031589397   7.190115020153727   3.856885000727702
7.090600941523509   3.476460666072041   3.727807248988862
3.601707961096307   3.648353082729004   0.199860166191001

ecut 200

