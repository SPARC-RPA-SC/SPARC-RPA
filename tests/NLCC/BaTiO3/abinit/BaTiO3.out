
.Version 8.10.3 of ABINIT 
.(MPI version, prepared for a x86_64_linux_gnu8.3 computer) 

.Copyright (C) 1998-2018 ABINIT group . 
 ABINIT comes with ABSOLUTELY NO WARRANTY.
 It is free software, and you are welcome to redistribute it
 under certain conditions (GNU General Public License,
 see ~abinit/COPYING or http://www.gnu.org/copyleft/gpl.txt).

 ABINIT is a project of the Universite Catholique de Louvain,
 Corning Inc. and other collaborators, see ~abinit/doc/developers/contributors.txt .
 Please read https://docs.abinit.org/theory/acknowledgments for suggested
 acknowledgments of the ABINIT effort.
 For more information, see https://www.abinit.org .

.Starting date : Wed 18 Nov 2020.
- ( at 02h53 )
  
- input  file    -> BaTiO3.in
- output file    -> BaTiO3.out
- root for input  files -> tbase_xi
- root for output files -> tbase_xo


 Symmetries : space group P1 (#  1); Bravais aP (primitive triclinic)
================================================================================
 Values of the parameters that define the memory need of the present run
     intxc =       0    ionmov =       0      iscf =       7    lmnmax =       6
     lnmax =       6     mgfft =     100  mpssoang =       3    mqgrid =    4858
     natom =       5  nloc_mem =       1    nspden =       1   nspinor =       1
    nsppol =       1      nsym =       1    n1xccc =    2501    ntypat =       3
    occopt =       7   xclevel =       1
-    mband =          29        mffmem =           1         mkmem =           1
       mpw =       30003          nfft =     1000000          nkpt =           1
================================================================================
P This job should need less than                     282.419 Mbytes of memory.
  Rough estimation (10% accuracy) of disk space for files :
_ WF disk file :     13.278 Mbytes ; DEN or POT disk file :      7.631 Mbytes.
================================================================================

--------------------------------------------------------------------------------
------------- Echo of variables that govern the present computation ------------
--------------------------------------------------------------------------------
-
- outvars: echo of selected default values                                      
-   iomode0 =  0 , fftalg0 =312 , wfoptalg0 =  0
-
- outvars: echo of global parameters not present in the input file              
-  max_nthreads =    0
-
 -outvars: echo values of preprocessed input variables --------
            acell      7.6300000000E+00  7.6300000000E+00  7.6300000000E+00 Bohr
              amu      1.37327000E+02  4.78800000E+01  1.59994000E+01
      chksymbreak           0
             ecut      2.00000000E+02 Hartree
-          fftalg         312
           istwfk        2
              ixc           2
         kptrlatt        1    0    0      0    1    0      0    0    1
          kptrlen      7.63000000E+00
P           mkmem           1
            natom           5
            nband          29
            ngfft         100     100     100
             nkpt           1
             nsym           1
           ntypat           3
              occ      2.000000  2.000000  2.000000  2.000000  2.000000  2.000000
                       2.000000  2.000000  2.000000  2.000000  2.000000  2.000000
                       2.000000  2.000000  2.000000  2.000000  2.000000  2.000000
                       2.000000  2.000000  0.000000  0.000000  0.000000  0.000000
                       0.000000  0.000000  0.000000  0.000000  0.000000
           occopt           7
            prtwf           0
          spgroup           1
           toldfe      1.00000000E-08 Hartree
           tsmear      7.34986400E-03 Hartree
            typat      1  2  3  3  3
           xangst      3.9902411689E+00  3.8359175980E+00  3.9786757106E+00
                       1.9715915973E+00  2.0523232946E+00  1.8134698373E+00
                       2.1446176571E+00  3.8048449958E+00  2.0409756385E+00
                       3.7521844135E+00  1.8396637510E+00  1.9726706342E+00
                       1.9059417650E+00  1.9306253003E+00  1.0576144485E-01
            xcart      7.5404630134E+00  7.2488337285E+00  7.5186074646E+00
                       3.7257681648E+00  3.8783289629E+00  3.4269613428E+00
                       4.0527400316E+00  7.1901150202E+00  3.8568850007E+00
                       7.0906009415E+00  3.4764606661E+00  3.7278072490E+00
                       3.6017079611E+00  3.6483530827E+00  1.9986016619E-01
             xred      9.8826513937E-01  9.5004373900E-01  9.8540071620E-01
                       4.8830513300E-01  5.0829999514E-01  4.4914303314E-01
                       5.3115858867E-01  9.4234797119E-01  5.0548951517E-01
                       9.2930549692E-01  4.5563049359E-01  4.8857237864E-01
                       4.7204560434E-01  4.7815898856E-01  2.6193992948E-02
            znucl       56.00000   22.00000    8.00000

================================================================================

 chkinp: Checking input parameters for consistency.

================================================================================
== DATASET  1 ==================================================================
-   nproc =    1

 Exchange-correlation functional for the present dataset will be:
  LDA: Perdew-Zunger-Ceperley-Alder - ixc=2
 Citation for XC functional:
  J.P.Perdew and A.Zunger, PRB 23, 5048 (1981)

 Real(R)+Recip(G) space primitive vectors, cartesian coordinates (Bohr,Bohr^-1):
 R(1)=  7.6300000  0.0000000  0.0000000  G(1)=  0.1310616  0.0000000  0.0000000
 R(2)=  0.0000000  7.6300000  0.0000000  G(2)=  0.0000000  0.1310616  0.0000000
 R(3)=  0.0000000  0.0000000  7.6300000  G(3)=  0.0000000  0.0000000  0.1310616
 Unit cell volume ucvol=  4.4419495E+02 bohr^3
 Angles (23,13,12)=  9.00000000E+01  9.00000000E+01  9.00000000E+01 degrees

 getcut: wavevector=  0.0000  0.0000  0.0000  ngfft= 100 100 100
         ecut(hartree)=    200.000   => boxcut(ratio)=   2.05871

--- Pseudopotential description ------------------------------------------------
- pspini: atom type   1  psp file is ./../../../../psps_LDA/Ba.psp8
- pspatm: opening atomic psp file    ./../../../../psps_LDA/Ba.psp8
- Ba    ONCVPSP-3.3.1  r_core=   2.15444   1.91218   2.15444
- 56.00000  10.00000    201113                znucl, zion, pspdat
    8    2    2    4       600   0.00000      pspcod,pspxc,lmax,lloc,mmax,r2well
    5.99000000000000    5.00000000000000    0.00000000000000   rchrg,fchrg,qchrg
     nproj     2     2     2
     extension_switch     1
  pspatm : epsatm=   25.96060710
         --- l  ekb(1:nproj) -->
             0    6.667782    0.685450
             1    5.045793    0.410143
             2    2.550865    0.419177
 pspatm: atomic psp has been read  and splines computed

- pspini: atom type   2  psp file is ./../../../../psps_LDA/Ti.psp8
- pspatm: opening atomic psp file    ./../../../../psps_LDA/Ti.psp8
- Ti    ONCVPSP-3.3.1  r_core=   1.35825   1.31050   1.66360
- 22.00000  12.00000    201113                znucl, zion, pspdat
    8    2    2    4       600   0.00000      pspcod,pspxc,lmax,lloc,mmax,r2well
    5.99000000000000    2.50000000000000    0.00000000000000   rchrg,fchrg,qchrg
     nproj     2     2     2
     extension_switch     1
  pspatm : epsatm=   14.76477602
         --- l  ekb(1:nproj) -->
             0   11.217743    1.241170
             1    1.919421   -0.083914
             2   -5.274126   -1.504205
 pspatm: atomic psp has been read  and splines computed

- pspini: atom type   3  psp file is ./../../../../psps_LDA/O.psp8
- pspatm: opening atomic psp file    ./../../../../psps_LDA/O.psp8
- O     ONCVPSP-3.3.1  r_core=   1.60139   1.60139
-  8.00000   6.00000    201113                znucl, zion, pspdat
    8    2    1    4       400   0.00000      pspcod,pspxc,lmax,lloc,mmax,r2well
    3.99000000000000    4.00000000000000    0.00000000000000   rchrg,fchrg,qchrg
     nproj     2     2
     extension_switch     1
  pspatm : epsatm=    8.41098440
         --- l  ekb(1:nproj) -->
             0    2.150546   -0.316620
             1   -4.646291   -1.320741
 pspatm: atomic psp has been read  and splines computed

   2.63833345E+03                                ecore*ucvol(ha*bohr**3)
--------------------------------------------------------------------------------

_setup2: Arith. and geom. avg. npw (full set) are   60005.000   60005.000

================================================================================

     iter   Etot(hartree)      deltaE(h)  residm     vres2
 ETOT  1  -135.79320053222    -1.358E+02 2.829E-01 2.560E+04
 ETOT  2  -125.46714587397     1.033E+01 3.480E-02 4.391E+05
 ETOT  3  -136.63529034218    -1.117E+01 3.166E-02 6.590E+03
 ETOT  4  -136.67399556479    -3.871E-02 2.731E-03 3.598E+03
 ETOT  5  -136.73091946964    -5.692E-02 4.667E-04 9.370E+02
 ETOT  6  -136.74296419787    -1.204E-02 3.853E-04 3.390E+02
 ETOT  7  -136.74789862155    -4.934E-03 5.799E-05 8.105E+01
 ETOT  8  -136.74947422405    -1.576E-03 5.823E-04 5.236E-01
 ETOT  9  -136.74947740154    -3.177E-06 2.413E-05 4.970E-01
 ETOT 10  -136.74948628076    -8.879E-06 1.208E-04 5.569E-02
 ETOT 11  -136.74948728143    -1.001E-06 6.626E-06 5.610E-03
 ETOT 12  -136.74948738890    -1.075E-07 5.800E-05 7.981E-05
 ETOT 13  -136.74948739118    -2.285E-09 3.539E-06 1.373E-06
 ETOT 14  -136.74948739120    -1.879E-11 3.231E-05 2.425E-07

 At SCF step   14, etot is converged : 
  for the second time, diff in etot=  1.879E-11 < toldfe=  1.000E-08

 Cartesian components of stress tensor (hartree/bohr^3)
  sigma(1 1)= -9.11007391E-04  sigma(3 2)=  7.05978998E-05
  sigma(2 2)= -9.81817929E-04  sigma(3 1)= -1.03719486E-04
  sigma(3 3)= -1.36819301E-03  sigma(2 1)=  2.01733771E-04

 Integrated electronic density in atomic spheres:
 ------------------------------------------------
 Atom  Sphere_radius  Integrated_density
    1        2.00000          5.31571286
    2        2.00000          9.64902068
    3        2.00000          6.35230227
    4        2.00000          6.33610757
    5        2.00000          6.40660847
================================================================================

 ----iterations are completed or convergence reached----

 Mean square residual over all n,k,spin=   11.143E-07; max=  32.313E-06
 reduced coordinates (array xred) for    5 atoms
       0.988265139367      0.950043738997      0.985400716198
       0.488305132996      0.508299995136      0.449143033135
       0.531158588675      0.942347971187      0.505489515167
       0.929305496923      0.455630493587      0.488572378635
       0.472045604338      0.478158988562      0.026193992948
 rms dE/dt=  6.4949E-01; max dE/dt=  1.3504E+00; dE/dt below (all hartree)
    1       0.026055319094     -0.012738382842      0.211916389748
    2      -0.727725715947     -1.407386863516      1.350389116554
    3      -0.090316763702      0.577661574040     -0.613446977670
    4       0.274811448802      0.621104082389     -0.444261231682
    5       0.517173788440      0.221360799066     -0.504596809952

 cartesian coordinates (angstrom) at end:
    1      3.99024116889213     3.83591759800365     3.97867571059565
    2      1.97159159728271     2.05232329457519     1.81346983734132
    3      2.14461765705743     3.80484499580598     2.04097563853773
    4      3.75218441346104     1.83966375104493     1.97267063418149
    5      1.90594176500932     1.93062530026926     0.10576144485329

 cartesian forces (hartree/bohr) at end:
    1     -0.00341490219611     0.00166954451762    -0.02777408811902
    2      0.09537684551566     0.18445440437008    -0.17698414405693
    3      0.01183700904835    -0.07570921785226     0.08039935453071
    4     -0.03601727830467    -0.08140286245893     0.05822560014171
    5     -0.06778167406322    -0.02901186857651     0.06613327750354
 frms,max,avg= 8.5123051E-02 1.8445440E-01   5.041E-08 -3.169E-08 -1.277E-08 h/b

 cartesian forces (eV/Angstrom) at end:
    1     -0.17560131652372     0.08585142368724    -1.42820091436286
    2      4.90447417718126     9.48502603760220    -9.10088983971749
    3      0.60868342729103    -3.89312418463257     4.13430068929221
    4     -1.85208276099489    -4.18590313739110     2.99408048989221
    5     -3.48547352695368    -1.49185013926577     3.40070957489593
 frms,max,avg= 4.3772029E+00 9.4850260E+00   2.592E-06 -1.630E-06 -6.564E-07 e/A
 length scales=  7.630000000000  7.630000000000  7.630000000000 bohr
              =  4.037622101542  4.037622101542  4.037622101542 angstroms
 prteigrs : about to open file tbase_xo_EIG
 Fermi (or HOMO) energy (hartree) =   0.25755   Average Vxc (hartree)=  -0.42734
 Eigenvalues (hartree) for nkpt=   1  k points:
 kpt#   1, nband= 29, wtk=  1.00000, kpt=  0.0000  0.0000  0.0000 (reduced coord)
  -1.80146   -0.94885   -0.94621   -0.94560   -0.71311   -0.36865   -0.34390   -0.33424
  -0.16668   -0.16586   -0.16550    0.14050    0.14773    0.15170    0.15940    0.16264
   0.18445    0.19758    0.20140    0.21571    0.32664    0.34142    0.35646    0.38430
   0.38622    0.41713    0.43051    0.46040    0.51389
      occupation numbers for kpt#   1
   2.00000    2.00000    2.00000    2.00000    2.00000    2.00000    2.00000    2.00000
   2.00000    2.00000    2.00000    2.00000    2.00000    2.00000    2.00000    2.00000
   2.00000    2.00000    2.00000    2.00000    0.00000    0.00000    0.00000    0.00000
   0.00000    0.00000    0.00000    0.00000    0.00000

--------------------------------------------------------------------------------
 Components of total free energy (in Hartree) :

    Kinetic energy  =  6.31272724894946E+01
    Hartree energy  =  3.27138822117689E+01
    XC energy       = -2.78344370980394E+01
    Ewald energy    = -1.00666304463461E+02
    PspCore energy  =  5.93958457040048E+00
    Loc. psp. energy= -1.02264956776384E+02
    NL   psp  energy= -7.76452832498180E+00
    >>>>> Internal E= -1.36749487391202E+02

    -kT*entropy     = -6.20742737001910E-17
    >>>>>>>>> Etotal= -1.36749487391202E+02

 Other information on the energy :
    Total energy(eV)= -3.72114279406022E+03 ; Band energy (Ha)=  -1.0677853148E+01
--------------------------------------------------------------------------------

 Cartesian components of stress tensor (hartree/bohr^3)
  sigma(1 1)= -9.11007391E-04  sigma(3 2)=  7.05978998E-05
  sigma(2 2)= -9.81817929E-04  sigma(3 1)= -1.03719486E-04
  sigma(3 3)= -1.36819301E-03  sigma(2 1)=  2.01733771E-04

-Cartesian components of stress tensor (GPa)         [Pressure=  3.1981E+01 GPa]
- sigma(1 1)= -2.68027583E+01  sigma(3 2)=  2.07706157E+00
- sigma(2 2)= -2.88860759E+01  sigma(3 1)= -3.05153211E+00
- sigma(3 3)= -4.02536215E+01  sigma(2 1)=  5.93521144E+00

== END DATASET(S) ==============================================================
================================================================================
  
 -outvars: echo values of variables after computation  --------
            acell      7.6300000000E+00  7.6300000000E+00  7.6300000000E+00 Bohr
              amu      1.37327000E+02  4.78800000E+01  1.59994000E+01
      chksymbreak           0
             ecut      2.00000000E+02 Hartree
           etotal     -1.3674948739E+02
            fcart     -3.4149021961E-03  1.6695445176E-03 -2.7774088119E-02
                       9.5376845516E-02  1.8445440437E-01 -1.7698414406E-01
                       1.1837009048E-02 -7.5709217852E-02  8.0399354531E-02
                      -3.6017278305E-02 -8.1402862459E-02  5.8225600142E-02
                      -6.7781674063E-02 -2.9011868577E-02  6.6133277504E-02
-          fftalg         312
           istwfk        2
              ixc           2
         kptrlatt        1    0    0      0    1    0      0    0    1
          kptrlen      7.63000000E+00
P           mkmem           1
            natom           5
            nband          29
            ngfft         100     100     100
             nkpt           1
             nsym           1
           ntypat           3
              occ      2.000000  2.000000  2.000000  2.000000  2.000000  2.000000
                       2.000000  2.000000  2.000000  2.000000  2.000000  2.000000
                       2.000000  2.000000  2.000000  2.000000  2.000000  2.000000
                       2.000000  2.000000  0.000000  0.000000  0.000000  0.000000
                       0.000000  0.000000  0.000000  0.000000  0.000000
           occopt           7
            prtwf           0
          spgroup           1
           strten     -9.1100739111E-04 -9.8181792869E-04 -1.3681930144E-03
                       7.0597899842E-05 -1.0371948575E-04  2.0173377059E-04
           toldfe      1.00000000E-08 Hartree
           tsmear      7.34986400E-03 Hartree
            typat      1  2  3  3  3
           xangst      3.9902411689E+00  3.8359175980E+00  3.9786757106E+00
                       1.9715915973E+00  2.0523232946E+00  1.8134698373E+00
                       2.1446176571E+00  3.8048449958E+00  2.0409756385E+00
                       3.7521844135E+00  1.8396637510E+00  1.9726706342E+00
                       1.9059417650E+00  1.9306253003E+00  1.0576144485E-01
            xcart      7.5404630134E+00  7.2488337285E+00  7.5186074646E+00
                       3.7257681648E+00  3.8783289629E+00  3.4269613428E+00
                       4.0527400316E+00  7.1901150202E+00  3.8568850007E+00
                       7.0906009415E+00  3.4764606661E+00  3.7278072490E+00
                       3.6017079611E+00  3.6483530827E+00  1.9986016619E-01
             xred      9.8826513937E-01  9.5004373900E-01  9.8540071620E-01
                       4.8830513300E-01  5.0829999514E-01  4.4914303314E-01
                       5.3115858867E-01  9.4234797119E-01  5.0548951517E-01
                       9.2930549692E-01  4.5563049359E-01  4.8857237864E-01
                       4.7204560434E-01  4.7815898856E-01  2.6193992948E-02
            znucl       56.00000   22.00000    8.00000

================================================================================


- Timing analysis has been suppressed with timopt=0



================================================================================

 Suggested references for the acknowledgment of ABINIT usage.

 The users of ABINIT have little formal obligations with respect to the ABINIT group
 (those specified in the GNU General Public License, http://www.gnu.org/copyleft/gpl.txt).
 However, it is common practice in the scientific literature,
 to acknowledge the efforts of people that have made the research possible.
 In this spirit, please find below suggested citations of work written by ABINIT developers,
 corresponding to implementations inside of ABINIT that you have used in the present run.
 Note also that it will be of great value to readers of publications presenting these results,
 to read papers enabling them to understand the theoretical formalism and details
 of the ABINIT implementation.
 For information on why they are suggested, see also https://docs.abinit.org/theory/acknowledgments.
-
- [1] Recent developments in the ABINIT software package.
- Computer Phys. Comm. 205, 106 (2016).
- X.Gonze, F.Jollet, F.Abreu Araujo, D.Adams, B.Amadon, T.Applencourt,
- C.Audouze, J.-M.Beuken, J.Bieder, A.Bokhanchuk, E.Bousquet, F.Bruneval
- D.Caliste, M.Cote, F.Dahm, F.Da Pieve, M.Delaveau, M.Di Gennaro,
- B.Dorado, C.Espejo, G.Geneste, L.Genovese, A.Gerossier, M.Giantomassi,
- Y.Gillet, D.R.Hamann, L.He, G.Jomard, J.Laflamme Janssen, S.Le Roux,
- A.Levitt, A.Lherbier, F.Liu, I.Lukacevic, A.Martin, C.Martins,
- M.J.T.Oliveira, S.Ponce, Y.Pouillon, T.Rangel, G.-M.Rignanese,
- A.H.Romero, B.Rousseau, O.Rubel, A.A.Shukri, M.Stankovski, M.Torrent,
- M.J.Van Setten, B.Van Troeye, M.J.Verstraete, D.Waroquier, J.Wiktor,
- B.Xu, A.Zhou, J.W.Zwanziger.
- Comment : the fourth generic paper describing the ABINIT project.
- Note that a version of this paper, that is not formatted for Computer Phys. Comm. 
- is available at https://www.abinit.org/about/ABINIT16.pdf .
- The licence allows the authors to put it on the Web.
- DOI and bibtex : see https://docs.abinit.org/theory/bibliography/#gonze2016
-
- [2] Optimized norm-conserving Vanderbilt pseudopotentials.
- D.R. Hamann, Phys. Rev. B 88, 085117 (2013).
- Comment: Some pseudopotential generated using the ONCVPSP code were used.
- DOI and bibtex : see https://docs.abinit.org/theory/bibliography/#hamann2013
-
- [3] ABINIT : First-principles approach of materials and nanosystem properties.
- Computer Phys. Comm. 180, 2582-2615 (2009).
- X. Gonze, B. Amadon, P.-M. Anglade, J.-M. Beuken, F. Bottin, P. Boulanger, F. Bruneval,
- D. Caliste, R. Caracas, M. Cote, T. Deutsch, L. Genovese, Ph. Ghosez, M. Giantomassi
- S. Goedecker, D.R. Hamann, P. Hermet, F. Jollet, G. Jomard, S. Leroux, M. Mancini, S. Mazevet,
- M.J.T. Oliveira, G. Onida, Y. Pouillon, T. Rangel, G.-M. Rignanese, D. Sangalli, R. Shaltaf,
- M. Torrent, M.J. Verstraete, G. Zerah, J.W. Zwanziger
- Comment : the third generic paper describing the ABINIT project.
- Note that a version of this paper, that is not formatted for Computer Phys. Comm. 
- is available at https://www.abinit.org/about/ABINIT_CPC_v10.pdf .
- The licence allows the authors to put it on the Web.
- DOI and bibtex : see https://docs.abinit.org/theory/bibliography/#gonze2009
-
- [4] A brief introduction to the ABINIT software package.
- Z. Kristallogr. 220, 558-562 (2005).
- X. Gonze, G.-M. Rignanese, M. Verstraete, J.-M. Beuken, Y. Pouillon, R. Caracas, F. Jollet,
- M. Torrent, G. Zerah, M. Mikami, Ph. Ghosez, M. Veithen, J.-Y. Raty, V. Olevano, F. Bruneval,
- L. Reining, R. Godby, G. Onida, D.R. Hamann, and D.C. Allan.
- Comment : the second generic paper describing the ABINIT project. Note that this paper
- should be cited especially if you are using the GW part of ABINIT, as several authors
- of this part are not in the list of authors of the first or third paper.
- The .pdf of the latter paper is available at https://www.abinit.org/about/zfk_0505-06_558-562.pdf.
- Note that it should not redistributed (Copyright by Oldenburg Wissenshaftverlag,
- the licence allows the authors to put it on the Web).
- DOI and bibtex : see https://docs.abinit.org/theory/bibliography/#gonze2005
-
- And optionally:
-
- [5] First-principles computation of material properties : the ABINIT software project. 
- X. Gonze, J.-M. Beuken, R. Caracas, F. Detraux, M. Fuchs, G.-M. Rignanese, L. Sindic,
- M. Verstraete, G. Zerah, F. Jollet, M. Torrent, A. Roy, M. Mikami, Ph. Ghosez, J.-Y. Raty, D.C. Allan.
- Computational Materials Science 25, 478-492 (2002). http://dx.doi.org/10.1016/S0927-0256(02)00325-7
- Comment : the original paper describing the ABINIT project.
- DOI and bibtex : see https://docs.abinit.org/theory/bibliography/#gonze2002
-
- Proc.   0 individual time (sec): cpu=         64.5  wall=         64.7

================================================================================

 Calculation completed.
.Delivered   0 WARNINGs and   4 COMMENTs to log file.
+Overall time at end (sec) : cpu=         64.5  wall=         64.7
